server/views/formPages/addingStaff/staff-involved.html

```diff
@@ -19,10 +19,14 @@ <h1 class="govuk-heading-xl mainHeading">{{ pageTitle }}</h1>
             text: item.name
             },
             {
-            text: item.email
+              text: item.prison | default('None', true)
```

Comment:
by default `default` only applies the default when the value is `undefined`. You need to pass true as the second param to use the default when the value is any falsy object. 

In this case it can be null so we need to pass the flasg.